docs/_set-up-moodle-via-aaf-authentication/03-add-a-new-user.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Add a new user
+order: 3
+duration: 1
+---
+
+Once on the Site Administration page, click on 'Users'.
```

Comment:
dont think we need this one, same as keycloack, central is the identity provider so it will handler creating the user on login?

Manualyl creating them sorta defeats the purpose of an IdP